ocp_resources/datavolume.py

```diff
@@ -1,11 +1,20 @@
 # -*- coding: utf-8 -*-
 
+from ocp_resources.cdi_config import CDIConfig
 from ocp_resources.constants import TIMEOUT_4MINUTES
 from ocp_resources.persistent_volume_claim import PersistentVolumeClaim
 from ocp_resources.resource import NamespacedResource, Resource
 from ocp_resources.utils import TimeoutExpiredError, TimeoutSampler
 
 
+LOGGER = get_logger(name=__name__)
+
+
+def is_garbage_collector_enabled_cdi_config():
+    dv_ttl_seconds = CDIConfig(name="config").instance.spec.get("dataVolumeTTLSeconds")
+    return not dv_ttl_seconds or dv_ttl_seconds >= 0
```

Comment:
According to https://github.com/kubevirt/containerized-data-importer/blob/main/doc/cdi-config.md#options:
```
Time in seconds after DataVolume completion it can be garbage collected. The default is 0 sec. To disable GC use -1.
```
So iiuc, `0` does not mean that GC is disabled

Reply:
you are right, I meant to the default that it is  not exists and equal to 0, please see change